src/js/utils/obj.js

```diff
@@ -30,3 +31,55 @@ export function reduce(object, fn, initial = 0) {
     (accum, key) => fn(accum, object[key], key),
     initial);
 }
+
+/**
+ * Object.assign-style object shallow merge/extend.
+ *
+ * @param  {Object} target
+ * @param  {Object} ...sources
+ * @return {Object}
+ */
+export function assign(target, ...sources) {
```

Comment:
How does that work if the target is the first argument and the sources are the rest?